pkgs/development/tools/icestorm/default.nix

```diff
@@ -1,19 +1,19 @@
-{ stdenv, fetchFromGitHub, python3, libftdi }:
+{ libcxxStdenv, fetchFromGitHub, python3, libftdi }:
 
-stdenv.mkDerivation rec {
+libcxxStdenv.mkDerivation rec {
```

Comment:
Is it really necessary to use libcxxStdenv? No other package in nixpkgs does that.